frontend/src/components/Registration.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Registration from './Registration';
+import { Router as Router, Route } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+const history = createMemoryHistory({
+    initialEntries: ['/']
+  });
+
+
+let utils =  ()=>{return render(
+    <Router history={history}>
+        <Route path = '/'>
+            <Registration />
+        </Route>
+    </Router>

```

Comment:
@zvorsi this is not really necessary to put it into the router, you can just render a `Registration` component to test

Reply:
at first it passes, then running test again fails, I would keep it that way